db/structure.sql

```diff
@@ -685,7 +685,7 @@ CREATE TABLE public.oban_jobs (
     attempted_by text[],
     discarded_at timestamp without time zone,
     priority integer DEFAULT 0 NOT NULL,
-    tags text[] DEFAULT ARRAY[]::text[],
+    tags character varying(255)[] DEFAULT ARRAY[]::character varying[],
```

Comment:
I just recreated my DB from phoenix and I think this field type is now text[] (meaning you can revert and not commit this change when it shows up in your diff), although I'm not sure if it makes much difference for the apps. It will just be churning back and forth depending on each devs local schema. Not a blocker for now, but we will eventually want to get the local dev schemas in sync.

Reply:
I went ahead and reverted this change on the branch